app/server_test.go

```diff
@@ -201,3 +205,66 @@ func checkEndpoint(t *testing.T, client *http.Client, url string, expectedStatus
 
 	return nil
 }
+
+func TestPanicLog(t *testing.T) {
+	// Creating a temp file to collect logs
+	tmpfile, err := ioutil.TempFile("", "mlog")
+	if err != nil {
+		require.NoError(t, err)
+	}
+
+	defer func() {
+		require.NoError(t, tmpfile.Close())
+		require.NoError(t, os.Remove(tmpfile.Name()))
+	}()
+
+	// Creating logger to log to console and temp file
+	logger := mlog.NewLogger(&mlog.LoggerConfiguration{
+		EnableConsole: true,
+		ConsoleJson:   true,
+		EnableFile:    true,
+		FileLocation:  tmpfile.Name(),
+	})
+
+	// Creating a server with logger
+	s, err := NewServer(SetLogger(logger))
+	require.NoError(t, err)
+
+	// Route for just panicing
+	s.Router.HandleFunc("/panic", func(writer http.ResponseWriter, request *http.Request) {
+		panic("log this panic")
```

Comment:
Totally a nit, but I'd add a regular log line before the panic and then make sure that both are appearing in the file.

Reply:
@levb Pushed this change. Please let me know if you want me to do other changes for logging rest of the panics using approach that I mentioned on mattermost or mark this PR as done to let it be merged.